api/idamCheck.ts

```diff
@@ -0,0 +1,21 @@
+import { getConfigValue } from './configuration';
+import { SERVICES_IDAM_WEB } from './configuration/references';
+import { http } from './lib/http';
+import { Request } from 'express';
```

Comment:
import ordering is wrong here - express should be at top right?

Reply:
Changed it now